notifications/NotificationBase.cpp

```diff
@@ -117,6 +117,7 @@ bool CNotificationBase::SendMessageEx(
 		fText = CURLEncode::URLEncode(fText);
 	}
 
+	boost::mutex::scoped_lock SendMessageEx(SendMessageExMutex);
```

Comment:
Why the mutex as this is the only place it's being used ? (i'm not so keep about mutexes, and try to avoid them where possible)